Lib/unittest/main.py

```diff
@@ -273,6 +273,11 @@ def runTests(self):
             testRunner = self.testRunner
         self.result = testRunner.run(self.test)
         if self.exit:
-            sys.exit(not self.result.wasSuccessful())
+            if self.result.testsRun == 0:
+                sys.exit(5)
+            elif not self.result.wasSuccessful():
+                sys.exit(1)
+            else:
+                sys.exit(0)
```

Comment:
Let's either flip these so positive comes first:
```suggestion
            elif self.result.wasSuccessful():
                sys.exit(0)
            else:
                sys.exit(1)

```

Or keep the original one-liner:

```suggestion
            else:
                sys.exit(not self.result.wasSuccessful())

```

(Plus an intentional newline at the end so we have 2 blank lines after the class.